@here/harp-text-canvas/package.json

```diff
@@ -24,6 +24,7 @@
     },
     "devDependencies": {
         "@here/harp-geoutils": "^0.18.0",
+        "@here/harp-materials": "^0.18.0",
```

Comment:
This should be a real dependency, otherwise the typings would not work for the users of this package.
Ideally I would prefer to not have this dependency b/c the plan was to factor out text canvas as a standalone module. Maybe we should move it to harp utils.
Do you know why we have geoutils as a dev dependency?

Reply:
@ninok  thanks for the review.

> This should be a real dependency, otherwise the typings would not work for the users of this package.
sure will fix it.

> Ideally I would prefer to not have this dependency b/c the plan was to factor out text canvas as a standalone module. Maybe we should move it to harp utils.
If I move WebGL1RawShaderMaterial to harp-utils, harp-utils will have a dependency to three.js, which is even nastier.
The only alternative I can think of is to move only the glsl conversion functions to harp-utils, then duplicate the onBeforeCompile implementation directly in the text materials. What do you think?

> Do you know why we have geoutils as a dev dependency?
I'll remove it and check if it breaks anything.

Reply:
done